v2/timeutils/milli_test.go

```diff
@@ -8,13 +8,13 @@ import (
 )
 
 func Test_MillisecondsShouldBeWithinRealisticInterval(t *testing.T) {
-	assert.True(t, MillisecondsNow() > 1528030261000) // 2018-03-06
-	assert.True(t, MillisecondsNow() < 9999999999999) // 2286-11-20
+	assert.True(t, MillisecondsNow() > 1528030261000) // nolint: gomnd
+	assert.True(t, MillisecondsNow() < 9999999999999) // nolint: gomnd
 }
 
 func Test_MillisecondsUnix(t *testing.T) {
 	now := time.Now()
-	assert.Equal(t, MillisecondsUnix(now.Add(time.Second)), MillisecondsUnix(now)+1000)
+	assert.Equal(t, MillisecondsUnix(now.Add(time.Second)), MillisecondsUnix(now)+1000) // nolint: gomnd
```

Comment:
+ 1 * time.second?

Reply:
Where do you believe I should add or change to `1 * time.Second`?

Reply:
instead of +1000 but I guess the linter would complain on the one. I think your solution is good here.